build/deps/build_deps.jsonc

```diff
@@ -93,6 +93,15 @@
       "repo": "rules_pkg",
       "file_regex": "\\.tar\\.gz$"
     },
+    {
+      // Needed for running the latest Node.js due to aspect_rules_js
```

Comment:
Nit: rules_js is not making the situation worse per se – we'd pull in an outdated version using `rules_js_dependencies()` without this change, but if we weren't using rules_js we'd still have to import it ourselves. This comment should say something like "due to aspect_rules_js using older version by default".

Reply:
Thanks! I updated the coment.